Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/internal/Hole_filling/Triangulate_hole_polygon_mesh.h

```diff
@@ -95,7 +95,9 @@ triangulate_hole_polygon_mesh(PolygonMesh& pmesh,
             VertexPointMap vpmap,
             bool use_delaunay_triangulation,
             const Kernel& k,
-            const bool use_cdt)
+            const bool use_cdt,
+            const typename Kernel::FT max_squared_distance =
+            typename Kernel::FT(4) / typename Kernel::FT(10000))
```

Comment:
the default should definitely be related to the input, using the bbox of the polyline for example. These days I'm working on models with a bbox of 1e-07 for example.

Reply:
I see. You are right.

Reply:
actually the default value here is not needed, right? only the call matters.

Reply:
Yes, her not really, but I need a default for the named parameters anyway.